source/slang/slang-capabilities.capdef

```diff
@@ -397,3 +448,83 @@ alias DX_6_4 = sm_6_4;
 alias DX_6_5 = sm_6_5;
 alias DX_6_6 = sm_6_6;
 alias DX_6_7 = sm_6_7;
+
+alias sm_2_0_GLSL_140 = sm_4_0 | glsl | spirv_1_0 | cuda | cpp;
+alias sm_2_0_GLSL_400 = sm_4_0 | glsl | spirv_1_0 | cuda | cpp;
+alias appendstructuredbuffer = sm_5_0 + shader_stages_fra_com;
+alias atomic_hlsl = _sm_4_0 | GLSL_430_SPIRV_1_0;
+alias atomic_hlsl_nvapi = _sm_4_0 + hlsl_nvapi;
+alias atomic_hlsl_sm_6_6 = _sm_6_6;
+alias byteaddressbuffer = sm_4_0;
+alias byteaddressbuffer_rw = sm_4_0 + shader_stages_fra_com;
+alias consumestructuredbuffer = sm_5_0 + shader_stages_fra_com;
+alias fragmentprocessing = fragment + sm_5_0;
+alias fragmentprocessing_derivativecontrol = fragment + sm_5_0 + GL_ARB_derivative_control;
+alias getattributeatvertex = fragment + _sm_6_1 | fragment + GL_EXT_fragment_shader_barycentric;
+alias memorybarrier_com = compute + sm_5_0;
```

Comment:
what is "com"? What does it stand for? compute?

Reply:
for shader stages since we can have very long names, I for-now used _com for compute, _fra for fragment, _ray for raygen, _any for anyhit, etc.

I welcome any suggestions to improve the naming to keep it short and concise.

Reply:
"com" and "any" could mean too many things. I don't see why we cannot use the full words like "_compute", "_fragment" and "_anyhit".

Another aspect is that if the name is distinctive, it is easy to replace the words with "vim" or any text replacing features. If the words are too short and common like "com" or "any", it will be difficult to replace them later.

Reply:
One of my rules for naming, for your information, is that when the scope of the variable is short, I go with short names. When the scope is larger, I use more descriptive and longer names.